.travis.yml

```diff
@@ -7,9 +7,10 @@ install:
   - npm install
 
 script:
-  - npm run vscode:prepublish
+  - gulp package
```

Comment:
Is there any difference between this and running "vsce package" from command line?

Reply:
`vsce package` requires you to have vsce installed globally on your machine. If you we do it through gulp, we can specify 'vsce' as a devDependency in package.json and it will use vsce from the repo's 'node_modules' folder (which is a more scalable pattern for CI systems)